### ISSUE TITLE
Failed to load the language-geck package

### ISSUE THREAD

**Issue by nyteshade** [nyteshade/language-geck#2]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [language-geck](https://github.com/nyteshade/language-geck) package, v0.3.0
### Stack Trace

Failed to load the language-geck package

```
At Unexpected token [

SyntaxError: Unexpected token [
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:77:42)
    at Object.defineProperty.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:715:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:117:28
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:106:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:457:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:536:30)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:452:29
    at exit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:73:16)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:215:47)
    at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:237:18)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```
### Commands

```
     -0:42.9.0 fuzzy-finder:toggle-file-finder (atom-pane.pane.active)
     -0:40.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:36.6.0 core:close (atom-text-editor.editor.is-focused)
     -0:16.2.0 settings-view:open (atom-pane.pane.active)
     -0:11.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{}
```
### Installed Packages

``` coffee
# User
language-geck, v0.3.0

# Dev
No dev packages
```

**Comment by Evolution0** [comment]
Should be fixed by #3 now but wouldn't want to assume as I'm not 100% sure that this isn't an issue in OSX only.

**Comment by nyteshade** [comment]
Thanks for your help. I’ll check on both. But to be fair, it’s unlikely given no OS X support for the games and how poorly they run in wine or emulators, that tons of folks are interested in copying over scripts from GECK to look at, edit and then move back to their mods.
